### ISSUE TITLE
Problems with event triggering in `ui-collapsible`

### ISSUE THREAD

**Issue by didimitrie** [JosephusPaye/Keen-UI#76]
Possibly my bad; but i'm desperately trying to update the height of all ui-collapsible components when i reload data dynamically.

I'm using `self.$broadcast('ui-collapsible::refresh-height')` to no avail! I am doing something wrong? Or do I need to pass the id of the collapsible element?

**Comment by JosephusPaye** [comment]
Hey @didimitrie,

Are you triggering the event immediately after changing the data? Since Vue updates the DOM asynchronously, it may be that the new content isn't in the DOM when the collapsible receives the event and updates its height.

Try using [`$nextTick`](http://vuejs.org/api/#vm-nextTick) and see if it works:

```js
// Update the instance data

this.$nextTick(function() {
    this.$broadcast('ui-collapsible::refresh-height');
});
```

As for passing an `id`, do that if you want to update only a specific collapsible.

**Comment by JosephusPaye** [comment]
May also be related to #97, which was fixed in #101

**Comment by JosephusPaye** [comment]
Should be fixed in [v0.8.9](https://github.com/JosephusPaye/Keen-UI/releases/tag/v0.8.9), try it out.

**Comment by didimitrie** [comment]
Will give it a shot soon - did some workarounds of my own, will revert back to the component when i have the time, let's see how it goes. 

as always, great work!
